src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $cart-border-color;
+  border-radius: 5px;
+  background-color: $cart-bg-color;
+  padding: $card-padding;
+
+  &__product-img {
+    display: flex;
+    justify-content: center;
+    padding: $product-img-padding 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-weight: 500;
+    line-height: 18px;
+    color: $page-text-color;
+    margin: $product-img-padding + 8 0 4px;
+  }
```

Comment:
The font size is missing. It's always a good practice to specify the font size for text elements.